extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -1029,17 +974,16 @@ public function isFieldSortable( $field ) {
 	 * @param string $field: field name
 	 * @param mixed $value: field value
 	 *
-	 * @return string: formated table field
+	 * @return string: formatted table field
 	 */
 	function formatValue( $field, $value ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 10, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)